.github/workflows/ci.yml

```diff
@@ -11,7 +11,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        ruby: [ '2.5', '2.6', '2.7', '3.0', '3.1' ]
+        ruby: [ '2.5', '2.6', '2.7', '3.0', '3.1', '3.2' ]
```

Comment:
I wonder if we should drop 2.5 and 2.6.

Reply:
RuboCop itself tests against 2.6 upwards:

https://github.com/rubocop/rubocop/blob/master/.circleci/config.yml

Reply:
The upcoming end-of-life of Ruby 2.7 might be a good time to drop everything older than Ruby 3, although that might be a little aggressive.